docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -48,7 +48,7 @@
             <td><h5>branch</h5></td>
```

Comment:
Maybe it is better to just add `ExcludeFromDocumentation` to branch option.

Reply:
Good idea~